gamification/process/update.py

```diff
@@ -135,6 +136,29 @@ def get_participant_objects():
     return participants
 
 
+def update_participants_data_with_meta_review(participants):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    recieved or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = meta_review_participants.objects.get(
+                        pos_in=participants.pos_in)
+    pos_out_meta_review = meta_review_participants.objects.get(
+                        pos_out=participants.pos_out)
+    neg_in_meta_review = meta_review_participants.objects.get(
+                        neg_in=participants.neg_in)
+    neg_out_meta_review = meta_review_participants.objects.get(
+                        neg_out=participants.neg_out)
+    if pos_in_meta_review > 0 or pos_out_meta_review > 0 or
```

Comment:
See https://stackoverflow.com/questions/181530/styling-multi-line-conditions-in-if-statements